aspen/algorithms/website.py

```diff
@@ -146,9 +147,9 @@ def delegate_error_to_simplate(website, state, response, request=None, resource=
 
     if fspath is not None:
         request.original_resource = resource
-        if resource is not None and resource.is_bound:
+        if resource is not None and resource.default_media_type != website.media_type_default:
```

Comment:
a) It's weird to have `default_media_type` vs. `media_type_default`.
b) What's the point of this check?